Book-A-Desk.Api/Program.fs

```diff
@@ -0,0 +1,34 @@
+﻿open Microsoft.AspNetCore.Builder
+open Microsoft.AspNetCore.Hosting
+open Microsoft.Extensions.Hosting
+open Microsoft.Extensions.DependencyInjection
+open Giraffe
+
+open BookADesk.Api.HttpHandlers
+
+let webApp =
```

Comment:
Personally, I prefer routing to be in a different file to the .NET setup.

Reply:
@TomasMorton I extracted the routes to a separate file. Can you please review again?